src/plugins/explore/public/application/pages/logs/logs_page.tsx

```diff
@@ -199,15 +202,19 @@ export const LogsPage: React.FC<Partial<Pick<AppMountParameters, 'setHeaderActio
             minSize="260px"
             mode={['collapsible', { position: 'top' }]}
             paddingSize="none"
+            style={{ display: showDataSetFields ? 'block' : 'none' }}
```

Comment:
nit. avoid inline styles

Reply:
i tried getting joey to change it but he said it shouldn't stop the bug fix. lets catch all these when merging to main